src/app/connections/list-page/list-page.component.spec.ts

```diff
@@ -1,39 +1,32 @@
 /* tslint:disable:no-unused-variable */
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+import { DebugElement } from '@angular/core';
 import { RouterTestingModule } from '@angular/router/testing';
-import { MockBackend } from '@angular/http/testing';
-import { RequestOptions, BaseRequestOptions, Http } from '@angular/http';
-import { RestangularModule } from 'ng2-restangular';
 
 import { IPaaSCommonModule } from '../../common/common.module';
-import { ConnectionsListPage } from './list-page.component';
 import { ConnectionsListComponent } from '../list/list.component';
+import { ConnectionsListPageComponent } from './list-page.component';
 import { ConnectionsListToolbarComponent } from '../list-toolbar/list-toolbar.component';
-import { StoreModule } from '../../store/store.module';
 
-describe('ConnectionListPage', () => {
-  let component: ConnectionsListPage;
-  let fixture: ComponentFixture<ConnectionsListPage>;
+describe('ConnectionsListComponent', () => {
```

Comment:
This would have the wrong output description now that it's been renamed, but I'd prefer not to rename it at all as I said previously.